modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic/screen.dm

```diff
@@ -0,0 +1,13 @@
+/datum/action/innate/monitor_change
+	name = "Screen Change"
+	check_flags = AB_CHECK_CONSCIOUS
+	icon_icon = 'icons/mob/actions/actions_silicon.dmi'
+	button_icon_state = "drone_vision"
+
+/datum/action/innate/monitor_change/Activate()
+	var/mob/living/carbon/human/H = owner
+	var/new_ipc_screen = input(usr, "Choose your character's screen:", "Monitor Display") as null|anything in GLOB.sprite_accessories[MUTANT_SYNTH_SCREEN]
+	if(!new_ipc_screen)
+		return
+	H.dna.species.mutant_bodyparts[MUTANT_SYNTH_SCREEN][MUTANT_INDEX_NAME] = new_ipc_screen
+	H.update_body()
```

Comment:
Convert this to a tgui_input_list, also single-letter variables bad :(

Reply:
What do you mean, single letter vars bad? I can read this code just fine right now, and I'll definitely be able to understand it one week later!

/s